evm-bytecode/EVMBParser.g4

```diff
@@ -0,0 +1,173 @@
+parser grammar EVMBParser;
+
+@ header
+{
+	package it.unipr.evm.antlr;
+}
```

Comment:
Also remove.

Reply:
Done in 762802de87efbd06bc511032c2ecc531751ad2b5